lib/streamlit/MediaFileManager.py

```diff
@@ -94,18 +94,18 @@ class MediaFileManager(object):
     def __init__(self):
         self._files = {}
         self._session_id_to_file_ids = collections.defaultdict(
-            set
-        )  # type: typing.DefaultDict[str, typing.Set[str]]
+            lambda: {}
+        )  # type: typing.DefaultDict[str, dict[str]]
```

Comment:
Shouldn't it be `typing.Dict[str, something]`?

Reply:
It maps to a file_id rather than a MediaFile